tests/integration-tests/tests/storage/test_shared_home.py

```diff
@@ -123,8 +123,8 @@ def _check_shared_home(
             scheduler_commands, remote_command_executor, f"{mount_dir}/{get_username_for_os(os)}"
         )
     elif storage_type == "Ebs":
-        _test_ebs_correctly_mounted(remote_command_executor, mount_dir, volume_size=40)
-        _test_ebs_correctly_mounted(remote_command_executor_login_node, mount_dir, volume_size=40)
+        _test_ebs_correctly_mounted(remote_command_executor, mount_dir, volume_size=45)
```

Comment:
Can you add change for this line too https://github.com/aws/aws-parallelcluster/pull/6725?

Reply:
not needed, we reverted the default shared ebs size to 40